addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -6,6 +6,10 @@
     display: flex;
     flex-direction: row;
     justify-content: space-between;
+    // We need the +1 to handle the status bar border-bottom.
+    // The var is called `$o-statusbar-height`, but is used on button, therefore
+    // doesn't include the border-bottom.
+    height: $o-statusbar-height + 1;
```

Comment:
Should be `min-height` because in mobile the topbar might go over 2 lines due to missing horizontal space for all buttons.
(add comment to explain it too)